src/llrb-tree/LlrbTree.ts

```diff
@@ -175,6 +176,115 @@ export class LlrbTree<K, V> implements SonicMap<K, V, LlrbNode<K, V>> {
     }
   }
 
+  // Helper methods for LLRB deletion
+  private _rotateLeft(node: LlrbNode<K, V>): LlrbNode<K, V> {
+    return rotateLeft(node, (newRoot) => {
```

Comment:
Do not use callbacks. This library cares for performance. Return the new root, if needed:

```suggestion
    this.root = rotateLeft(...
```

See, how it is done for reference in plain Red-Black trees in `src/red-black/util.ts`.